lib/tutorial.js

```diff
@@ -21,22 +21,22 @@ define([], function () {
 				<h3 class='popover-title tutorial-title'></h3>\
 				<div class='popover-content'></div>\
 				<div class='popover-navigation'>\
-					<div class='tutorial-prev-icon icon-button' data-role='prev'>\
+					<div align="left" class='tutorial-prev-icon icon-button' data-role='prev'>\
```

Comment:
You can't add double-quote in a double-quoted string. It broke the syntax of the file.